### PR TITLE
Added more predefined systems

### PR DESCRIPTION
**Pull request by KalelR**
Hi. I've added the following systems:

Continuous:
Hodgkin-Huxley
Hindmarsh-rose
Forced Van der Pol oscillator
Lotka-volterra predator-prey model
Stuart-Landau oscillator.

Discrete:
Tent map
Beta transformation 
Rulkov map
Ikeda map

I've checked the codes comparing eg the generated trajectories with figures from other sources. I didn't calculate the Jacobian for the Hodgkin-Huxley model because it is a bit complicated. Also, there may be formatting mistakes in the documentation, as I don't know how to visualize its compiled version : ( 

If anything is needed,  just let me know :)

### CODE REVIEW

File: src/predefined/continuous_famous_systems.jl [hunk 1]
```diff
@@ -1038,4 +1038,224 @@ function sprott_dissipative_conservative_J(u, p, t)
...
+activity (action potentials) in neurons. A complete description of all parameters and variables is given in [1][2][3]. The equations and default parameters used here are taken from [2][3]. They differ slightly from the original paper [1], since they were changed to shift the resting potential to -65 mV, instead of the 0mV in the original paper. 
+
+Varying the injected current I from `I = -5`  to  `I = 12` takes the neuron from quiescent to a single spike, and to a tonic (repetitive) spiking. This is due to a subcritical Hopf bifurcation, which occurs close to `I = 9.5`.
+
+[1] : A. L. Hodgkin, A.F. Huxley J. Physiol., pp. 500-544 (1952).
+[2] : G. Bard Ermentrout, and David H. Terman, "Mathematical Foundations of Neuroscience", Springer (2010).
+[3] : L. F. Abbott, and P. Dayan, "Theoretical Neuroscience: Computational and Mathematical Modeling of Neural Systems", MIT Press (2005).
+"""
+
+#In Ermentrout's & Abbott's books
```
Comment (Datseris) [inline src/predefined/continuous_famous_systems.jl:1071]: there cannot be empty space between the docstring and the function decleration.

**Comment by Datseris** [comment]
To see the docstrings: use Juno (atom) or VSCode and load the documentation pane. It displays docstrings with markdown rendering

File: src/predefined/continuous_famous_systems.jl [hunk 2]
```diff
@@ -1038,4 +1038,224 @@ function sprott_dissipative_conservative_J(u, p, t)
...
+        du4 = αh*(1-h) - βh*h
+        return SVector{4}(du1, du2, du3, du4)
+    end
+end
+
+
+
+"""
+```julia
+forcedvanderpol(u0=[0.5, 0.0]; μ=1.5, F=1.2, T=10) -> ds
```
Comment (Datseris) [inline src/predefined/continuous_famous_systems.jl:1099]: I would call this just `vanderpol`

**Comment by Datseris** [comment]
Seems great to me! Can you please increment minor version in Project.toml and also add a changelog entry saying something like "added many new famous systems"?

**Comment by KalelR** [comment]
> Can you please increment minor version in Project.toml
Sure! It is currently at v2.2.1. Do I increment it to v2.2.2, or v2.3?

**Comment by Datseris** [comment]
v2.3

**Comment by KalelR** [comment]
Done :)

**Pull request merged**